.github/workflows/prep-release.yml

```diff
@@ -0,0 +1,42 @@
+name: 'Step 1: Prep Release'
+on:
+  workflow_dispatch:
+    inputs:
+      version_spec:
+        description: 'New Version Specifier'
+        default: 'next'
+        required: false
+      branch:
+        description: 'The branch to target'
+        required: false
+      post_version_spec:
+        description: 'Post Version Specifier'
+        required: false
+      since:
+        description: 'Use PRs with activity since this date or git reference'
+        required: false
+      since_last_stable:
+        description: 'Use PRs with activity since the last stable git tag'
+        required: false
+        type: boolean
+jobs:
+  prep_release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: jupyterlab/maintainer-tools/.github/actions/base-setup@v1
+
+      - name: Prep Release
+        id: prep-release
+        uses: jupyter-server/jupyter_releaser/.github/actions/prep-release@v2
+        with:
+          token: ${{ secrets.ADMIN_GITHUB_TOKEN }}
```

Comment:
Since we don't have a bot for the `plasmabio` org, wondering if we should try to use a GitHub app instead, as suggested by this change in the releaser repo: https://github.com/jupyter-server/jupyter_releaser/pull/557

Reply:
Done, could you create an app for the `plasmabio` org?

Reply:
Let me check, but not sure I have admin rights on the org.

Reply:
No I don't have access to the organization settings at the moment.

cc @pierrepo who might be able to help with this. For context we are trying to setup automated releases for `tljh-repo2docker` using the Jupyter Releaser.

Reply:
It should be ok @jtpio :

![image](https://github.com/plasmabio/tljh-repo2docker/assets/733861/69093a54-62d2-4684-959b-de36e9d243fb)

Reply:
Nice, thanks!